File: 8KYU/Difference of Volumes of Cuboids/solution.cs
using System.Linq;
public class Kata
{
	public static int FindDifference(int[] a, int[] b)
	{ 
		var vola = a.Aggregate(1, (acc, val) => acc * val);
		var volb = b.Aggregate(1, (acc, val) => acc * val);

		return (vola > volb)
				  ? vola - volb
				  : volb - vola;
	}
}

File: 8KYU/Difference of Volumes of Cuboids/solution.c
int volume(const int c[3]) {
  return c[0] * c[1] * c[2];
}

int find_difference(const int a[3], const int b[3]) {
    int av = volume(a);
    int bv = volume(b);
    return abs(av-bv);
}